src/sidebar/store/create-store.ts

```diff
@@ -1,6 +1,6 @@
 /* global process */
 import * as redux from 'redux';
-import thunk from 'redux-thunk';
+import { thunk } from 'redux-thunk';
```

Comment:
Using the `thunk` named export now has the right types and `redux-thunk/extend-redux` is no longer needed (in fact it is no longer included)